config/environments/test.rb

```diff
@@ -35,6 +35,7 @@
 
   # Tests assume English text on the site.
   config.i18n.default_locale = "en"
+  I18n.locale = config.i18n.locale = config.i18n.default_locale
```

Comment:
This file is loaded after `config/application.rb`, where this this trick happens to properly set the locales in delayed_jobs, see 01bf64e6b03658424b9cd547dc6dc2cccdda669f

I just duplicated the line from `config/application.rb` here.